client/my-sites/purchases/index.js

```diff
@@ -172,21 +142,27 @@ export default ( router ) => {
 	router(
 		'/purchases/:siteName/:purchaseId/payment/add',
 		( { params: { siteName, purchaseId } } ) =>
-			isEnabled( 'purchases/new-payment-methods' )
-				? page.redirect(
-						`/purchases/subscriptions/${ siteName }/${ purchaseId }/payment-method/add`
-				  )
-				: page.redirect( `/purchases/subscriptions/${ siteName }/${ purchaseId }/payment/add` )
+			page.redirect( `/purchases/subscriptions/${ siteName }/${ purchaseId }/payment-method/add` )
 	);
 	router(
 		'/purchases/:siteName/:purchaseId/payment/edit/:cardId',
 		( { params: { siteName, purchaseId, cardId } } ) =>
-			isEnabled( 'purchases/new-payment-methods' )
-				? page.redirect(
-						`/purchases/subscriptions/${ siteName }/${ purchaseId }/payment-method/change/${ cardId }`
-				  )
-				: page.redirect(
-						`/purchases/subscriptions/${ siteName }/${ purchaseId }/payment/edit/${ cardId }`
-				  )
+			page.redirect(
+				`/purchases/subscriptions/${ siteName }/${ purchaseId }/payment-method/change/${ cardId }`
+			)
+	);
+	router(
```

Comment:
Similar to my above comment: I'm surprised that we had legacy routes here that have been disabled since the flag was introduced... are we sure they aren't covered elsewhere?